.github/CONTRIBUTING.md

```diff
@@ -106,82 +105,83 @@ Possible log levels are:`trace`, `debug`, `info`, `warn`, `err`, `critical`, `of
 
 ### Adding a new Traversal
 * Create a new traversal class under `src/autopas/containers/[Container]/traversals` for the container the traversal is intended.
-* Think about inheriting from a similar traversal. At least derive your new traversal from `src/autopas/containers/cellPairTraversals/TraversalInterface.h`.
-* Go to `src/autopas/options/TraversalOption.h`.
+* Think about inheriting from a similar traversal. At least derive your new traversal from [`TraversalInterface`](/src/autopas/containers/TraversalInterface.h).
+* Go to [`TraversalOption`](/src/autopas/options/TraversalOption.h).
   * Add a new enum in `TraversalOption::Value`.
   * Add a new string representation in the `map` of `TraversalOption::getOptionNames()`.
-* Add the enum to every compatible container in `src/autopas/containers/CompatibleTraversals.h`.
+* Add the enum to every compatible container in [`CompatibleTraversals`](/src/autopas/containers/CompatibleTraversals.h).
   * If applicability of the traversal is restricted, add your new enum to any of the functions that return sets of restricted traversals (E.g. `CompatibleTraversals::allTraversalsSupportingOnlySoA()`).
-* Add a case for the new traversal in `src/autopas/tuning/selectors/TraversalSelector.h::generateTraversal()`.
+* Add a case for the new traversal in [`TraversalSelector::generateTraversal()`](/src/autopas/tuning/selectors/TraversalSelector.h).
 * Check that the new option is working in the md-flexible example.
-* Adapt unit tests (e.g. expected number of iterations in `tests/testAutopas/tests/tuning/AutoTunerTest.cpp::testAllConfigurations()` and `OptionTest::parseTraversalOptionsTest`).
+* Adapt unit tests (e.g. expected number of iterations in [`AutoTunerTest::testAllConfigurations()`](/tests/testAutopas/tests/tuning/AutoTunerTest.cpp) and [`OptionTest::parseTraversalOptionsTest`](/tests/testAutopas/tests/options/OptionTest.h)).
 * Add new unit tests for your traversal.
-* Regenerate the `RuleLanguage.g4` via the `generateRuleLanguage.sh` script, both located in `src/autopas/tuning/tuningStrategy/ruleBasedTuning`
+* Regenerate the [`RuleLanguage.g4`](/src/autopas/tuning/tuningStrategy/ruleBasedTuning/RuleLanguage.g4) via the [`generateRuleLanguage.sh`](/src/autopas/tuning/tuningStrategy/ruleBasedTuning/generateRuleLanguage.sh) script, both located in [`ruleBasedTuning`](/src/autopas/tuning/tuningStrategy/ruleBasedTuning).
 
 ### Adding a new Container
-* Create a new container class under `src/autopas/containers/`.
-* Derive your new container from `src/autopas/containers/ParticleContainerInterface.h` or a more similar one.
-* Go to `src/autopas/options/ContainerOption.h`.
+* Create a new container class under [`src/autopas/containers/`](/src/autopas/containers/).
```

Comment:
As we are removing links from text everywhere, we could use something like `folder: containers`. I am not sure, just a suggestion. 

If seems feasible, also to be done at few other places.

Reply:
can you post an example what exactly you mean? Not sure I understand...

Reply:
I meant something like 'Create a new container class under the folder: [containers](src/autopas/containers/)'. But it also looks fine as it is.

Reply:
No, I think it is fine. In fact, it is more readable this way :)